indexing-service/src/main/java/org/apache/druid/indexing/common/TaskMetadata.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.druid.indexing.common;
```

Comment:
is this the correct package for this?

Reply:
Can a task operate on two datasources?

Reply:
ok i understand that it can now after chatting with some other people. Would it be clearer if i said TargetDataSource?

Reply:
Something like parseExceptionMetadata ?